public/sprites/brick-shrapnel.json

```diff
@@ -3,26 +3,26 @@
 
     "frames": [
         {
-            "name": "brick-1",
+            "style": "brick-1",
             "rect": [192, 168, 8, 8]
         },
         {
-            "name": "brick-2",
+            "style": "brick-2",
             "rect": [200, 168, 8, 8]
         },
         {
-            "name": "brick-3",
+            "style": "brick-3",
             "rect": [192, 176, 8, 8]
         },
         {
-            "name": "brick-4",
+            "style": "brick-4",
             "rect": [200, 176, 8, 8]
         }
     ],
 
     "animations": [
         {
-            "name": "spinning-brick",
+            "style": "spinning-brick",
```

Comment:
Animations should still have name as identifier.

Reply:
`animations` now named with `name`.